tests/run-with-compiler/tasty-hash.scala

```diff
@@ -0,0 +1,25 @@
+import dotty.tools.dotc.core.tasty.TastyHash.pjwHash64
+
+object Test {
```

Comment:
I would write a junit test instead, they're faster to run.

Reply:
Done